r2/r2/models/mail_queue.py

```diff
@@ -433,6 +435,7 @@ def utf8(s, reject_newlines=True):
             msg['To']      = utf8(self.to_addr)
             msg['From']    = utf8(fr)
             msg['Subject'] = utf8(self.subject)
+            msg['Date'] = utf8(email.utils.formatdate(time.mktime(self.date.timetuple())))
```

Comment:
sorry to nitpick, but this line is too long: https://github.com/reddit/styleguide/blob/master/python/pep-0008.rst#maximum-line-length